test/blocks/caas/caas.test.html

```diff
@@ -87,6 +99,16 @@
           expect(loadStyle.callCount).to.equal(1);
           expect(caasEl).to.equal(document.getElementById('caas'));
         });
+
+        it('it can use the v2 hash format', async () => {
+          const a = document.getElementById('v2-caaslink');
+          await init(a);
+          await waitForElement('#caas');
+          await delay(5);
```

Comment:
Shoul we use a fake timer instead? I'm fine with the delay as well if it's deterministic such as waiting for an eventListener execution and doesn't lead to flakiness.